src/data/pokemon/form_species_table_pointers.h

```diff
@@ -684,8 +684,12 @@ const u16 *const gFormSpeciesIdTables[NUM_SPECIES] =
     [SPECIES_LAPRAS_GIGANTAMAX] = sLaprasFormSpeciesIdTable,
     [SPECIES_EEVEE_GIGANTAMAX] = sEeveeFormSpeciesIdTable,
     [SPECIES_SNORLAX_GIGANTAMAX] = sSnorlaxFormSpeciesIdTable,
+#if P_GEN_5_POKEMON == TRUE
     [SPECIES_GARBODOR_GIGANTAMAX] = sGarbodorFormSpeciesIdTable,
+#endif
+#if P_GEN_7_POKEMON == TRUE
     [SPECIES_MELMETAL_GIGANTAMAX] = sMelmetalFormSpeciesIdTable,
+#endif
     [SPECIES_RILLABOOM_GIGANTAMAX] = sRillaboomFormSpeciesIdTable,
```

Comment:
I think it's missing #if P_GEN_8_POKEMON == TRUE

Reply:
Oh, apparently all Gmax forms are surrounded by `#if P_GEN_8_POKEMON == TRUE`.
I'll change that so it only groups the appropiate ones.

Reply:
Actually, I'm gonna move these to `gSpeciesInfo` anyway, and those are already grouped by family. Is it ok to skip this for now?